doc/Language/containers.pod6

```diff
@@ -351,18 +351,6 @@ The constraint is a property of the variable, not the container.
 In this case, the type constraint is the (compile-type defined) subset
 C<Three-letter>.
 
-Variables may have no container in them, yet still offer the ability to
-re-bind and typecheck that rebind. The reason for that is in such cases the
-binding operator L<:=|/language/operators#infix_:=> performs the typecheck:
-
-    my Int \z = 42;
-    z := 100; # OK
-    z := "x"; # Typecheck failure
```

Comment:
This returns now 

```
Cannot bind to 'z' because it is a term and terms cannot be rebound
```

So I guess we're good deleting this. But I'm not so fond myself of deleting stuff.  This was introduced here 94c2270cb3c35ff12b91e111b901a50b7e5ef9a6 Apparently that's no longer the case, but apparently also that definition is now creating a Term. So maybe a reference to it being a term (and thus, not rebindable) might be better than total elimination.